src/hb-subset-plan.hh

```diff
@@ -55,6 +60,16 @@ hb_subset_plan_new_gid_for_old_id(hb_subset_plan_t *plan,
                                   hb_codepoint_t old_gid,
                                   hb_codepoint_t *new_gid /* OUT */);
 
+HB_INTERNAL hb_bool_t
+hb_subset_plan_new_gid_for_codepoint(hb_subset_plan_t *plan,
```

Comment:
This, ideally, will be replaced by just a hashmap.

Reply:
Yup, there's a note on the plan about moving to a real map at which point several of these methods get put down with prejudice.